frontend/src/components/shared/RangeFilter.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <div class="is-flex">
     <input
-      v-model.number="min"
+      v-debounce:300ms="minChange"
+      :debounce-events="['click', 'keydown']"
       :class="{ 'input is-danger': !validMin }"
       :title="validMin ? '' : 'Please enter a valid number, e.g. 1.23'"
       class="vgt-input px-2 mr-1"
       type="number"
       placeholder="min"
     />
     <input
-      v-model.number="max"
+      v-debounce:300ms="maxChange"
```

Comment:
Could I ask more details on the rationale for this value for the delay, comparing to https://github.com/MetabolicAtlas/MetabolicAtlas/blob/365b6b159f6f445c619be6642c65c7ca27031bc9/frontend/src/components/explorer/gemBrowser/GemSearch.vue#L25

Reply:
I think 300ms should be fine here. In general I think this number should be as little as possible while achieving the effect we want (eliminate race condition). For the GEM search, I think on average the request takes longer (around 600ms from a couple of tests I did now compared to usually sub 100ms response times for enzymes table filtering) so I think it's ok to have slightly longer debounce time there. Wouldn't mind if we had a new issue to test this more in-depth but I think it's ok to have different values now.